### ISSUE TITLE
Compiling for pythonOCC installed with Conda

### ISSUE THREAD

**Issue by Puffton** [tpaviot/smesh#35]
Sorry, I must be misunderstanding something. The instructions say:

```bash
$ mkdir cmake-build
$ cd cmake-build
$ make
$ make install
```

So create an empty directory (where, in the smesh directory as fetched by git?) and then go in there and make? There will be nothing in the empty dir to make, or?

I tried the following:
```bash
$ mkdir Test
$ cd Test
$ git clone git://github.com/tpaviot/smesh.git
$ cd smesh
$ cmake CMakeLists.txt
$ make
$ make install
```

Then:
```python
>>> from OCC.SMESH import SMESH_Gen
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named SMESH
```

**Comment by tpaviot** [comment]
@Puffton You missed the cmake step, see https://github.com/tpaviot/pythonocc-core/blob/master/INSTALL.md

Can you point me to the file where you read those wrong instructions ?

**Comment by Puffton** [comment]
Thank you for your quick reply. I found it in the README on the front page: https://github.com/tpaviot/smesh

**Comment by tpaviot** [comment]
Fixed README by commit 2884fc9. Please submit pythonocc related issues to the pythonocc tracker.
